project/plugins.sbt

```diff
@@ -6,3 +6,4 @@ libraryDependencies += "org.scala-sbt" %% "scripted-plugin" % sbtVersion.value
 addSbtPlugin("com.typesafe.sbt" % "sbt-site" % "1.4.1")
 addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.6.3")
 addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
+addSbtPlugin("com.github.sbt" % "sbt-native-packager" % "1.9.7")
```

Comment:
This is for your personal use, or would we be expected to make installer releases?
If so I don't know if we want to get stuck with more work.

Reply:
@eed3si9n I'm planing to refresh `g8` on `macports` and it seems an easy way to build it on macports.

Reply:
ok. sounds good then.